staging/src/k8s.io/apimachinery/pkg/api/errors/errors.go

```diff
@@ -29,12 +29,6 @@ import (
 	"k8s.io/apimachinery/pkg/util/validation/field"
 )
 
-const (
-	// StatusTooManyRequests means the server experienced too many requests within a
-	// given window and that the client must wait to perform the action again.
-	StatusTooManyRequests = 429
```

Comment:
stripping this from staging area straightly breaks backward-compatibility

/hold

Reply:
make sense, fixed.